.github/workflows/tests.yml

```diff
@@ -16,6 +16,7 @@ jobs:
         with:
           node-version: 18.x
       - run: npm ci && npm run postinstall
+      - run: npm run prebuild
```

Comment:
Just to be clear, `prebuild` should automatically run when `npm run build` is triggered. But since we don't build before running our tests, I understand we have to manually call it here

Reply:
Yup! Didnt wanted to add a `npm run build` in here because its not needed by the test script